duckduckgo/macos-browser
# Bookmark button visible if url is bookmarked

Task/Issue URL: https://app.asana.com/0/1177771139624306/1202819750571708/f

**Description**:
Making the bookmark button in the address bar visible all the time in case the URL is bookmarked.

**Steps to test this PR**:
1. Verify the bookmark button is not visible in case URL is not bookmarked and mouse cursor located above the webview
2. Verify the bookmark button is visible in case: 
a) Mouse cursor is located above navigation bar and URL is not part of bookmarks
b) URL is part of bookmarks and mouse cursor is located above the webview



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
